static/sitemap/Country.0.txt

```diff
@@ -0,0 +1,267 @@
+https://datacommons.org/place?dcid=country/BDI
+https://datacommons.org/place?dcid=country/FXX
+https://datacommons.org/place?dcid=country/GUY
+https://datacommons.org/place?dcid=country/MSR
+https://datacommons.org/place?dcid=country/BYS
+https://datacommons.org/place?dcid=country/SDN
+https://datacommons.org/place?dcid=country/DJI
+https://datacommons.org/place?dcid=country/GHA
+https://datacommons.org/place?dcid=country/GNQ
+https://datacommons.org/place?dcid=country/HVO
+https://datacommons.org/place?dcid=country/PHL
+https://datacommons.org/place?dcid=iso/NL
```

Comment:
oh interesting:
https://datacommons.org/browser/iso/NL works
https://datacommons.org/place?dcid=iso/NL fails

searching for netherlands though, resolves to:
https://datacommons.org/place?dcid=country%2FNLD

looks like this could be an import error @tjann ?

Reply:
Added a fix in https://github.com/datacommonsorg/website/pull/153